src/workshop-comments.njk

```diff
@@ -0,0 +1,58 @@
+---
+pagination:
+    data: collections.workshops
+    size: 1
+    alias: workshopItem
+permalink: "initiatives/{{ workshopItem.id }}/"
+---
+{% extends 'layouts/base.njk' %}
+
+{% set pageTitle = workshopItem.title %}
+
+{% block content %}
+	<article class="workshop-page">
+		<h1>{{ workshopItem.title | safe }}</h1>
+		{% if workshopItem.imageUrl %}
+			<img class="workshop-image" src="{{ workshopItem.imageUrl | safe }}" alt="{{ workshopItem.imageAlt }}">
+		{% endif %}
+
+		<div class="description">{{ workshopItem.fullDescription | safe }}</div>
+
+		<form id="comment-form">
+			<h3>Comments</h3>
+			<div class="form-field">
+				<span>*</span><p class="asterisk-def"> Indicates required field</p>
+			</div>
+
+			<div class="form-field">
+				<span>*</span><label for="name"><h4>Name</h4></label><span class="required">Required</span>
+				<input id="name" type="text" name="name" :value="nameTerm" placeholder="" aria-label="Enter your name">
+			</div>
+
+			<div class="form-field">
+				<span>*</span><label for="comment"><h4>Post a comment or question</h4></label><span class="required">Required</span>
+				<textarea id="comment" type="text" name="comment" :value="commentTerm" placeholder="" aria-label="Enter your comment"></textarea>
+				<input type="hidden" name="workshopId" id="workshopId" value="{{ workshopItem.id }}">
+				<button id="post-comment" type="submit">Post a comment</button>
+			</div>
+		</form>
+
+		<div class="submitted-comment-message post-success-message" aria-live="assertive">
```

Comment:
While looking up for ARIA live region usage for status, I came across these ARIA roles explained by WCAG specs:

1. [Using role=status to present status messages](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA22.html)
2. [Using ARIA role=alert or Live Regions to Identify Errors](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA19.html)

These articles talk about live region attributes that are implied in these roles.

It feels to me the success message is more like a status message that could use `role=status` while the error message could `role=alert`. What do you think?